spec.md

```diff
@@ -402,6 +402,10 @@ The `<location>` is a pullable manifest URL.
 
 An attempt to pull a nonexistent repository MUST return response code `404 Not Found`
 
+A registry SHOULD enforce some limit on the maximum manifest size that it can accept.
```

Comment:
While this is recommended for public registries, I can see a case for private registries not wanting to implement this.

```suggestion
A registry MAY enforce limits on the maximum manifest size that it can accept.
```